spec/services/encryption/multi_region_kms_migration/profile_migrator_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe Encryption::MultiRegionKmsMigration::ProfileMigrator do
+  let(:profile) { create(:profile, :with_pii) }
+  let!(:user_password) { profile.user.password }
+  let!(:personal_key) { PersonalKeyGenerator.new(profile.user).normalize(profile.personal_key) }
+
+  subject { described_class.new(profile) }
+
+  before do
+    allow(IdentityConfig.store).to receive(:aws_kms_multi_region_read_enabled).and_return(true)
+  end
+
+  describe '#migrate!' do
+    context 'for a user without multi-region ciphertexts' do
```

Comment:
Relative to my comment above, is it worth adding specs for users that have one of the pii fields and not the other?

Reply:
A user should never have one of the PII fields and not the other. I just a pushed a commit to raise if we encounter that scenario.